### PR TITLE
Include reference to Honeycomb in tracing docs

### PR DESCRIPTION
**Pull request by spire-allyjweir**
Companion change to support https://github.com/concourse/concourse/pull/6103

### CODE REVIEW

File: lit/docs/operation/tracing.lit [hunk 1]
```diff
@@ -34,6 +34,15 @@ although support for other systems is planned to expand as the underlying SDK
...
   the default location that the \code{gcloud} CLI expects, or from GCP's
   metadata server (if Concourse is deployed on GCP).
+  
+  To export spans to Honeycomb.io, specify the API key, dataset and
+  optionally the service name:
+
+  \codeblock{bash}{{{
+  CONCOURSE_TRACING_HONEYCOMB_API_KEY: your-honeycomb-api-key
+  CONCOURSE_TRACING_HONEYCOMB_DATASET: your-honeycomb-dataset
+  CONCOURSE_TRACING_HONEYCOMB_SERVICE_NAME: service-name-for-concourse  # NOTE: Optional. Defaults to "concourse"
```
Comment (chenbh) [inline lit/docs/operation/tracing.lit:44]:
```suggestion
  CONCOURSE_TRACING_HONEYCOMB_API_KEY=your-honeycomb-api-key
  CONCOURSE_TRACING_HONEYCOMB_DATASET=your-honeycomb-dataset
  CONCOURSE_TRACING_HONEYCOMB_SERVICE_NAME=service-name-for-concourse  # NOTE: Optional. Defaults to "concourse"
```

**Review by chenbh — CHANGES_REQUESTED** [review]
Can you also update the blurb at the top to include honeycomb.io as well:
https://github.com/concourse/docs/blob/6c3bea059e8b8b48a22d7309a4124a2b30b910c2/lit/docs/operation/tracing.lit#L11-L14

File: lit/docs/operation/tracing.lit [hunk 2]
```diff
@@ -34,6 +34,15 @@ although support for other systems is planned to expand as the underlying SDK
   \link{\code{GOOGLE_APPLICATION_CREDENTIALS} environment variable}{https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable},
   the default location that the \code{gcloud} CLI expects, or from GCP's
   metadata server (if Concourse is deployed on GCP).
+
```
Comment (chenbh) [inline lit/docs/operation/tracing.lit:37]: I think the build failure is due to whitespaces on empty lines, I was able to build it locally after removing the 2 spaces on this line.

Comment (spire-allyjweir) [reply lit/docs/operation/tracing.lit:37]: I'll fix this up and the other suggestion. Thanks for the review!

**Comment by chenbh** [comment]
Hey, it looks like the build is still failing due to that whitespace on line 37. Once that and the DCOs are fixed this is good to be merged!

**Comment by spire-allyjweir** [comment]
Hi @chenbh, I hadn't spotted your response 9 days ago. I've fixed up the PR and hopefully it's ready for merge now.

**Review by chenbh — APPROVED** [review]
lgtm, thanks!

**Pull request merged**